pkg/operator/config/config.go

```diff
@@ -87,6 +87,8 @@ type OperatorConfig struct {
 
 	DisabledControllers []string `yaml:"disabledControllers"`
 
+	TeamStructureCSVFile string `yaml:"teamStructureCSVFile"`
```

Comment:
this is coming as ConfigMap mount generated from team tracking spreadsheet CSV export